components/build/.tekton/trigger-template.yaml

```diff
@@ -63,14 +73,53 @@ spec:
                     script: |
                       #!/usr/bin/env bash
                       MY_QUAY_USER=redhat-appstudio BUILD_TAG=$(params.revision) SKIP_BUILD=1 SKIP_INSTALL=1 hack/build-and-push.sh
+            - name: update-infra-repo
+              runAfter:
+                - build-bundles
+              params:
+                - name: BASE_IMAGE
+                  value: registry.redhat.io/openshift-pipelines/pipelines-git-init-rhel8@sha256:af7dd5b3b1598a980f17d5f5d3d8a4b11ab4f5184677f7f17ad302baa36bd3c1
+                - name: GIT_USER_NAME
+                  value: Build deployer
+                - name: GIT_SCRIPT
+                  value: |
+                    ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
+                    git clone https://github.com/redhat-appstudio/infra-deployments
+                    cd infra-deployments
+                    git remote add push git@github.com:Michkov/infra-deployments.git
```

Comment:
We shouldn't use personal accounts for such tasks

Reply:
Updated to new org redhat-appstudio-mr-creation